pdc/apps/changeset/middleware.py

```diff
@@ -34,9 +34,16 @@ def _may_announce_big_change(self, changeset, request):
                 len(changeset.tmp_changes) < settings.CHANGESET_SIZE_ANNOUNCE):
             return
 
+        author = str(changeset.author)
+        whitelist = getattr(settings, 'BIG_CHANGE_AUTHORS_WHITELIST', set())
+
```

Comment:
Here you get settings.BIG_CHANGE_AUTHORS_WHITELIST in every instance?
Like what's my concern in below comments.
Is it better to config the white list in a file? Then deploy by ansible in project. And you can get that file in every instance, and if it has changes the new data will be load, like remove/add people in whitelist. 
Although load file every time is not really good, but it can keep data fresh.